source/config/localhost_config.json

```diff
@@ -7,5 +7,10 @@
        "server_cert": "",
        "server_key": "",
        "root_cert": ""
+    },
+    "core_configuration": {
+       "sideband_read_write_core": 8,
```

Comment:
I think CpuCoreConfiguration or StreamingCoreConfiguration would be better

Reply:
renamed to `StreamingCoreConfiguration`